portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,6 +43,14 @@ public Comment(String email, String comment, String date){
       this.date = date;
     }
   }
+  private static class Response{
```

Comment:
Nit: there should be spaces before and after every function and internal class in the file (also applies at line 60 after init()).

Reply:
l got this one but couldn't figure out where you wanted the space after init()
Do you mean a new line?